### PR TITLE
Stas Hardening Oct 11

### PR DESCRIPTION
**Pull request by stas-vilchik**
Don't be afraid, tickets are really small :wink:

### CODE REVIEW

File: server/sonar-web/src/main/js/apps/background-tasks/components/Stats.js [hunk 1]
```diff
@@ -38,17 +40,17 @@ export default class Stats extends React.PureComponent {
...
+    event.preventDefault();
+    event.currentTarget.blur();
     this.props.onCancelAllPending();
-  }
+  };
 
-  handleShowFailing(e /*: Object */) {
-    e.preventDefault();
-    e.target.blur();
+  handleShowFailing = (event /*: Object */) => {
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/apps/background-tasks/components/Stats.js]: MINOR Remove parentheses around the parameter of this arrow function. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)

File: server/sonar-web/src/main/js/apps/background-tasks/components/Stats.js [hunk 2]
```diff
@@ -38,17 +40,17 @@ export default class Stats extends React.PureComponent {
   /*:: props: Props; */
   /*:: state: State; */
 
-  handleCancelAllPending(e /*: Object */) {
-    e.preventDefault();
-    e.target.blur();
+  handleCancelAllPending = (event /*: Object */) => {
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/apps/background-tasks/components/Stats.js]: MINOR Remove parentheses around the parameter of this arrow function. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)

File: server/sonar-web/src/main/js/apps/background-tasks/components/Workers.tsx [hunk 3]
```diff
@@ -64,16 +66,30 @@ export default class Workers extends React.PureComponent {
...
   };
 
+  handleHelpClick = (event: React.SyntheticEvent<HTMLAnchorElement>) => {
+    event.preventDefault();
+    event.stopPropagation();
+    this.toggleNoSupportPopup();
+  };
+
+  toggleNoSupportPopup = (show?: boolean) => {
+    if (show != undefined) {
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/apps/background-tasks/components/Workers.tsx]: MAJOR != should be !== [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1440)

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: server/sonar-web/scripts/start.js [hunk 4]
```diff
@@ -104,6 +105,9 @@ function runDevServer(compiler, host, port, protocol) {
     },
     setup(app) {
       app.use(errorOverlayMiddleware());
+      app.get('/api/l10n/index', (req, res) => {
```
Comment (gregaubert) [inline server/sonar-web/scripts/start.js]: Yes! That's great! :heart_eyes:

**Pull request merged**